torch/fx/experimental/symbolic_shapes.py

```diff
@@ -4411,7 +4411,11 @@ def _maybe_evaluate_static(
                 # Skip var_ranges logic for SingletonInt which is only used
                 # for jagged layout NestedTensors today
                 continue
-            vr = var_ranges[k]
+            try:
+                vr = self.var_to_range[k]
+            except KeyError:
+                log.warning(f"{k} is not in var_to_range, defaulting to unknown range.")
+                vr = ValueRanges(-sys.maxsize, sys.maxsize)
```

Comment:
Use `self._default_unspecified_value_range()` then, as this one has an off-by-one error.